docs/docusaurus.config.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {
+  title: 'Frigate',
+  tagline: 'NVR With Realtime Object Detection for IP Cameras',
+  url: 'https://blakeblackshear.github.io/frigate',
```

Comment:
Not sure if you're going to use the GitHub default or set a custom domain. I believe this should work as a default.